test/itkAttenuationImageFilterTest.cxx

```diff
@@ -48,7 +48,12 @@ itkAttenuationImageFilterTest(int argc, char * argv[])
   using MaskImageType = itk::Image<LabelType, Dimension>;
 
   SpectraImageType::Pointer inputImage = itk::ReadImage<SpectraImageType>(std::string(argv[1]));
-  MaskImageType::Pointer    maskImage = itk::ReadImage<MaskImageType>(std::string(argv[2]));
+  MaskImageType::Pointer    maskImage;
+  std::string               maskFilename = argv[2];
+  if (maskFilename != "nul")
```

Comment:
Nitpick, "nul" or "null"?

Reply:
`NUL`, thinking of Windows special file. I should have maybe spelled it with uppercase letters.

Reply:
Got it, thanks for the explanation.